episodes/chapter4.md

```diff
@@ -0,0 +1,242 @@
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+In this section, we will go through the steps to create a new organization, a new user, a new collaboration and reset an api key for a node using the Vantage6 UI.
+
+First you need to log in to the Vantage6 UI, and then go to the `Administration` page. You can find the `Administration` button in the top right corner of the start page.
+
+![Vantage6 UI administration page](fig/ui_admin_page.png)
+
+### Create a new organization
+It's quite straightforward to create a new organization in Vantage6. Here are the steps:
+
+1. Click on the `Organizations` tab in the administration page.
+2. Click on the `Create organization` button.
+3. Fill in the details of the new organization.
+    * The `Upload public key` field is optional. You can upload a public key for the organization if you want to use encryption in the collaboration.
+4. Click on the `Submit` button to create the new organization.
+
+![Create a new organization](fig/create_org.png)
+
+### Create a new user
+Now let's create a new user for the organization we just created. Here are the steps:
+
+1. Click on the `Users` tab in the administration page.
+2. Click on the `Create user` button.
+3. Fill in the details of the new user.
+    * You can assign the user to an organization by selecting it from the `Organization` dropdown. Only one organization can be selected.
+    * You can assign roles to the user by selecting them from the `Roles` dropdown. Here we assign the `Researcher` role to the new user. You can give the user more permissions by assigning multiple roles or select the operation boxes in the `Permissions` section.
+4. Click on the `Submit` button to create the new user.
+
+![Create a new user](fig/create_user.png)
+
+### Create a new collaboration
+A collaboration is a collection of one or more organizations. Let's create a new collaboration between two organizations. Here are the steps:
+
+1. Click on the `Collaborations` tab in the administration page.
+2. Click on the `Create collaboration` button.
+3. Fill in the details of the new collaboration.
+    * For `Encrypted` box, you can select whether the collaboration should be encrypted or not.
+    * You can select the organizations that will participate in the collaboration by selecting them from the `Organizations` dropdown.
+    * By default, we select the `Register nodes` box. This will ensure the nodes of the organizations are registered in the collaboration. If you don't select this box, you will have to register the nodes manually later.
+
+![Create a new collaboration](fig/create_collab_01.png)
+
+4. Click on the `Submit` button to create the new collaboration.
+    * After submitting the form, you will see a dialog box to ask you to download the API key. The API key is used to authenticate the nodes in the collaboration. Later we will see how to reset the API key for a node.
```

Comment:
The dialog will not ask you to download them but points out that they have been downloaded :-)

Reply:
Wait, I guess the difference is that on Windows I don't get this box asking me to set the filename, but is just downloads it
Too bad I'm stuck with windows

Reply:
Added the scenario for Windows.

Reply:
I added the scenario for Windows.